pkg/models/querybuilder_scene.go

```diff
@@ -10,7 +10,9 @@ import (
 	"github.com/stashapp/stash/pkg/database"
 )
 
-var scenesForPerformerQuery = selectAll("scenes") + `
+const TABLE_NAME = "scenes"
```

Comment:
Golang constants are visible module-wide, so `TABLE_NAME` is not a good constant name for a scene-specific value. Suggest changing this to `sceneTable` or similar.

Reply:
Duh, I keep forgetting that go is basically C, scope and all. Fixed.